frontend/javascripts/oxalis/model_initialization.ts

```diff
@@ -560,6 +576,7 @@ function determineDefaultState(
   const { viewMode } = temporaryConfiguration;
   const defaultPosition = datasetConfiguration.position;
```

Comment:
It should also be possible to configure the default `additionalCoordinates`. Could be part of a followup PR.

Reply:
I added this to https://github.com/scalableminds/webknossos/issues/7229.